src/ApiPort.VS2013/ApiPort.VS2013.csproj

```diff
@@ -0,0 +1,325 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename the project from ApiPort.VS2013 back to ApiPortVS since does not open on VS2013 anymore.